models/choral.js

```diff
@@ -61,8 +61,10 @@ choralSchema.statics.createNew = function (attrs, cb) {
     }
     else {
       cb(null, newChoral);
+      console.log('Device was created successfully');
     }
   });
+  return newChoral;
```

Comment:
If we do this, and use the returned value, the returned value might not contain anything yet, since the database queries are being done asynchronously (this could return before the database is done), and therefore, there could have been an error or something. It is better to pass a callback to `createNew` and then inside that callback, do what you need to do with the new choral. 

e.g. https://gist.github.com/cpjk/20cbb79d0cba0e33c49f5ae6fdf76dfb#file-choralcreatenew-js-L5-L11